modules/gis/src/lib/geojson-to-binary.js

```diff
@@ -1,13 +1,13 @@
 // Convert GeoJSON features to flat binary arrays
 export function geojsonToBinary(features, options = {}) {
   const firstPassData = firstPass(features);
-  return secondPass(features, firstPassData, options);
+  return secondPass(features, {...firstPassData, ...options});
```

Comment:
Nit: This is a bit too compact for my taste, as these are very different objects with just a small overlap of fields. I would more explicitly copy any auto deduced options into the options object:

```
return secondPass(features, firstPassData, {maxCoordLength: firstPassData.maxCoordLength, ..., ...options);
```

Being more strict with "types" will also help as we are gradually introducing typescript type definition files in this repo.

Reply:
What do you think about that revision? It's more explicit and essentially defines three options that a user can provide:

- `coordLength` force a specific coordinate length.
- `numericPropKeys` could pass `[]` to prevent creating any numeric property arrays
- `PositionDataType` could pass `Float64Array` to store positions at higher precision

Reply:
+1